### ISSUE TITLE
v9 Gui: Not able to activate some security events

### ISSUE THREAD

**Issue by fdurand** [inverse-inc/packetfence#4491]
- [ ] Security Event 1100004 : 
![image](https://user-images.githubusercontent.com/1553962/57703632-72511380-762e-11e9-832c-e035a8530dd4.png)

- [ ] Security Event 1100006:
![image](https://user-images.githubusercontent.com/1553962/57703741-a9bfc000-762e-11e9-9654-4fe7abb0f448.png)

- [ ] Security Event 1300002:
![image](https://user-images.githubusercontent.com/1553962/57704308-9fea8c80-762f-11e9-9b78-1a10df4f0d7c.png)

**Comment by cgx** [comment]
The backend can't resolve some the of the devices:

```
May 14 09:59:00 localhost.localdomain packetfence[7668]: INFO pfperl-api(7668): Could not find any 'Device' entry with ID '31' in schema 'Upstream' (fingerbank::Base::CRUD::read)
May 14 09:59:00 localhost.localdomain packetfence[7668]: ERROR pfperl-api(7668): Unable to read device 31 (Class::MOP::Class:::after)
May 14 09:59:01 localhost.localdomain packetfence[7668]: INFO pfperl-api(7668): Could not find any 'Device' entry with ID '32' in schema 'Upstream' (fingerbank::Base::CRUD::read)
May 14 09:59:01 localhost.localdomain packetfence[7668]: ERROR pfperl-api(7668): Unable to read device 32 (Class::MOP::Class:::after)
```

**Comment by jrouzierinverse** [comment]
Partially fixed by d700b939330300c14a93a4a6fbf7cc7f174ad6e7

**Comment by julsemaan** [comment]
Tried to toggle all the available default security events and they worked so we should be golden
